project_timesheet_hours/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Project Timesheet Hours
+=======================
+
+This module computes a progress rate for a project based on task planned hours
+and spent hours.
+
+Installation
+============
+
+Just install it.
+
+Configuration
+=============
+
+No configuration needed.
+
```

Comment:
Please add "usage" section and explain where we have to go for checking this new information. Thanks!